packages/react-scripts/template/src/index.css

```diff
@@ -1,5 +1,12 @@
+* {
+  box-sizing: border-box;
```

Comment:
It depends how involved you want it to be really..

The usual suspect resets are either:
- [normalize.css](https://github.com/necolas/normalize.css) about setting sensible defaults to reduce browser inconsistencies 
- [reset.css](http://meyerweb.com/eric/tools/css/reset/). more about stripping browser defaults leaving everything bare bones.

Alternatively we could extend on this to be something like:

```
* {
  box-sizing: border-box;
  margin: 0;
  padding: 0;
  border: 0;
}

body {
  font-family: sans-serif;
  font-size: 16px;
  line-height: 1;
}
```

Which achieves a pretty sufficient (if not slightly brutal) reset that is easy to comprehend and easy to extend. Without employing a dependency.

What do you think?